tests/tests/buffer.rs

```diff
@@ -217,15 +217,19 @@ static MINIMUM_BUFFER_BINDING_SIZE_LAYOUT: GpuTestConfiguration = GpuTestConfigu
                 push_constant_ranges: &[],
             });
 
-        wgpu_test::fail(&ctx.device, || {
-            ctx.device
-                .create_compute_pipeline(&wgpu::ComputePipelineDescriptor {
-                    label: None,
-                    layout: Some(&pipeline_layout),
-                    module: &shader_module,
-                    entry_point: "main",
-                });
-        });
+        wgpu_test::fail(
+            &ctx.device,
+            || {
+                ctx.device
+                    .create_compute_pipeline(&wgpu::ComputePipelineDescriptor {
+                        label: None,
+                        layout: Some(&pipeline_layout),
+                        module: &shader_module,
+                        entry_point: "main",
+                    });
+            },
+            None,
```

Comment:
- [ ] todo: I'd like to ask for others' thoughts on taking the time to narrow the accepted validation error message in tests with this PR. I think a separate change to update each call to `wgpu_test::fail` to actually specify an expected validation message and maybe even force `expected_msg_substring` to be required programatically would be good, but it doesn't _have_ to be now (and I don't consider it a personal priority.

CC @jimblandy, @cwfitzgerald, @teoxoy, @nical.

Reply:
I'm gonna punt forcing concrete validation error messages for now. Maybe a `good first issue`? 🙂